### ISSUE TITLE
 OSError: [Errno 22] Invalid argument

### ISSUE THREAD

**Issue by galantra** [open-spaced-repetition/fsrs4anki-helper#27]
When I enter the review screen:

```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue to the add-on author.
Debug info:
Anki 2.1.60 (76d88073) Python 3.9.15 Qt 5.15.2 PyQt 5.15.5
Platform: Windows-10-10.0.19045
Flags: frz=True ao=True sv=3
Add-ons, last update check: 2023-02-25 20:34:44

Caught exception:
Traceback (most recent call last):
  File "aqt.main", line 1139, in onStudyKey
  File "aqt.main", line 715, in moveToState
  File "_aqt.hooks", line 4506, in __call__
  File "C:\Users\username\AppData\Roaming\Anki2\addons21\673114053\__init__.py", line 94, in maybe_restore_sidebar
    cs.show()
  File "C:\Users\username\AppData\Roaming\Anki2\addons21\673114053\sidebar_base.py", line 125, in show
    self.setup_style()
  File "C:\Users\username\AppData\Roaming\Anki2\addons21\673114053\sidebar_base.py", line 59, in setup_style
    self.set_day_style()
  File "C:\Users\username\AppData\Roaming\Anki2\addons21\673114053\sidebar_base.py", line 81, in set_day_style
    update_contents_of_sidebar(self)
  File "C:\Users\username\AppData\Roaming\Anki2\addons21\673114053\sidebar_set_contents.py", line 58, in update_contents_of_sidebar
    p = current_card_deck_properties(card)
  File "C:\Users\username\AppData\Roaming\Anki2\addons21\673114053\card_deck_properties.py", line 51, in current_card_deck_properties
    p["c_Due"] = due_day(card)
  File "C:\Users\username\AppData\Roaming\Anki2\addons21\673114053\helper_functions.py", line 46, in due_day
    return time.strftime("%Y-%m-%d", time.localtime(mydue))
OSError: [Errno 22] Invalid argument

```

**Comment by L-M-Sherlock** [comment]
673114053 is not the FSRS4Anki helper add-on.

**Comment by galantra** [comment]
Sorry for bothering you! I was in a hurry and didn't check the addon number. I had just updated FSRS4Anki, so I was under the impression that it was related.

**Comment by galantra** [comment]
Just some context in case anyone gets the same error: 

1. I had just updated scheduler_qt5.js (from 3.13.4 to 3.14.3, if I recall correctly)
2. Then I rescheduled cards all cards with the Helper.
3. Then I got the above error and the [sidebar from a certain add-on ](https://ankiweb.net/shared/info/673114053) didn't load
4. There were just a handful of cards affected by this
5. On the same cards where the sidebar didn't load, `display_memory_state` in FSRS4Anki showed `R: 0.00%`
